contrib/devtools/security-check.py

```diff
@@ -116,6 +117,21 @@ def check_ELF_control_flow(binary) -> bool:
         return True
     return False
 
+def check_ELF_FORTIFY(binary) -> bool:
+
+    chk_funcs = set()
+
+    for sym in binary.symbols:
+        match = re.search(r'__[a-z]*_chk', sym.name)
```

Comment:
Might want to check `.imported` to make sure it's an imported symbol, just to be sure.

Reply:
Added.